qiskit/algorithms/optimizers/scipy_minimizer.py

```diff
@@ -0,0 +1,100 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Powell optimizer."""
+
+from collections.abc import Sequence
+from typing import Any, Dict, Optional, Union
+
+from scipy.optimize import Bounds, minimize
+
+from .optimizer import Optimizer, OptimizerSupportLevel
+
+
+class ScipyMinimizer(Optimizer):
+    """
+    Optimizer using scipy.optimize.minimize.
+
+    For further detail, please refer to
+    See https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize.html
+    """
+
+    def __init__(
+        self,
+        method: str,
+        options: Optional[Dict[str, Any]] = None,
+        max_evals_grouped: int = 1,
```

Comment:
I don't think we can support `max_evals_grouped` for this class. How can we tell scipy's `minimize` function that it can give several function values to the objective?